plugins/GSdx/GSRendererHW.cpp

```diff
@@ -45,23 +45,27 @@ GSRendererHW::GSRendererHW(GSTextureCache* tc)
 		m_userhacks_align_sprite_X = 0;
 	}
 
+		m_fb_width = max((int)m_context->FRAME.FBW * 64, 512);
```

Comment:
I know that my previous comment wasn't very clear. Sorry.

Actually, I think we can only cache the m_width/m_height variable. Besides, I'm afraid that it won't be legal to read context information here (aka the init).

So you can move back those lines in SetScaling. This way variable can be local.